tldw_Server_API/tests/Chatbooks/test_chatbook_service.py

```
@@ -160,6 +160,29 @@ def test_create_export_job(self, service, mock_db):
+    def test_get_media_db_uses_shared_factory_and_caches_result(self, mock_db, tmp_path, monkeypatch):
+        """Media DB lookup should use the shared factory and cache the instance."""
+        monkeypatch.setenv('PYTEST_CURRENT_TEST', 'test')
+        monkeypatch.setenv('USER_DB_BASE_DIR', str(tmp_path))
+        mock_db.execute_query.return_value = []
+
+        service = ChatbookService(user_id="test_user", user_id_int=7, db=mock_db)
+        fake_media_db = MagicMock()
+
+        with patch(
+            'tldw_Server_API.app.core.Chatbooks.chatbook_service.DatabasePaths.get_media_db_path',
+            return_value=Path('/tmp/media.db'),
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/media.db"

(S108)